.github/workflows/test-workflow.yml

```diff
@@ -237,14 +238,14 @@ jobs:
 
       - name: Build Gateway
         run: |
-          cargo build
+          cargo build --release --features freeze-time --features runtime-debug
```

Comment:
Note: we should probably use `--features integration` which properly includes sub-features.

Reply:
Addressing in my tweaks branch.